tools/CoqMakefile.in

```diff
@@ -308,8 +308,8 @@ else
 endif
 
 ifneq (,$(PROFILING))
-  PROFILE_ARG=-profile $<.prof.json
-  PROFILE_ZIP=gzip -f $<.prof.json
+  PROFILE_ARG=-profile $@.prof.json
```

Comment:
This probably now deserves a changelog like "Passing `PROFILING=1` to `make` with a `coq_makefile` `Makefile` now outputs profiling information to `foo.vo.prof.json.gz` instead of `foo.v.prof.json.gz`.  Additionally, `make validate` now supports `PROFILING=1` and outputs profiling information to `validate.prof.json.gz`"